src/exports/index.ts

```diff
@@ -5,14 +5,20 @@ export type {
   DeployOptions,
   DeploySentTx,
 } from "../contract-deploy.js";
+export { Contract } from "../contract.js";
```

Comment:
Wasn't able to use the `Contract` class without this.

Reply:
this is not exported in the index because due to Nemi internal monorepo limitations. Please import using `import { Contract } from "@nemi-fi/wallet-sdk/eip1193"`